Qt.py

```diff
@@ -881,7 +1180,13 @@ def _setup(module, extras):
             submodule = _import_sub_module(
                 module, name)
         except ImportError:
-            continue
+            try:
```

Comment:
This allows support for keeping sip/shiboken around in Qt.
I am using this for the changes in the getCppPointer unified function above.
If you look at L1389  for example, we are adding "shiboken" and "sip" to the extras that get passed into this.
This part allows us to bind it to Qt._sip and Qt._shiboken.

Reply:
Under PySide Qt._sip doesn't exist. It just wouldn't be an attribute.  The extras list is specific to each binding's setup function.

This was just support for stuff that used those binding specific modules to defer the actual function call to the unified _wrapinstance for example

Reply:
Technically linters are going to get mad at QtPyConvert using any private underscore things from Qt.py

So I think this is an acceptable solution for QtPyConvert if you are ok with it.

Reply:
I agree. Hound is great though, as it keeps Qt.py tidy. Unfortunately, it doesn't seem like you can tell hound to ignore certain linting errors: https://github.com/houndci/hound/issues/1184

Reply:
I can make it a double underscore if we'd like, however it might make more sense to think about 
"an api" to develop on top of. @mottosso brought this up in the gitter chat briefly.

Would you be ok if we kept it private for now and once we decide about some sort of Qt.py api we can discuss moving it then?